project/.gitlab-ci.yml

```diff
@@ -47,6 +51,9 @@ check-dependencies:
   <<: *install-deps
   stage: tests
   script: poetry run duty test
+  artifacts:
+    paths:
+    - .coverage-*
```

Comment:
Keep generated coverage files. For Python 3.6, coverage file is named `.coverage-36`, etc.
See `duties.py`, actions `coverage` and `test`.